src/appmixer/brevo/lib.generated.js

```
@@ -0,0 +1,124 @@
+        return '';
+    }
+
+    return [
+        headers.join(','),
+
+        ...array.map(items => {
+            return Object.values(items).map(property => {
+                if (typeof property === 'object') {
+                    return JSON.stringify(property);
+                }
+                return property != null ? property : '';;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)